### PR TITLE
Added Report Error Popup

### PR DESCRIPTION
**Pull request by john-baute**
Added Report Error popup to the Create Report screen when report creation fails.

### CODE REVIEW

File: CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx [hunk 1]
```diff
@@ -88,15 +93,60 @@ export default function CreateReport({ onDone }) {
...
+        top: '50%', 
+        left: '50%', 
+        transform: 'translate(-50%, -50%)',
+        background: 'white',
+        padding: '20px',
+        zIndex: 100,
+        width: 'auto',
+        textAlign: 'center',
+        borderRadius: '5px',
+      }}>
```
Comment (waffy1901) [inline CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx:132]: We should be using tailwind instead of regular css here

Comment (john-baute) [reply CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx:132]: fixing now

**Review by Carson21 — APPROVED** [review]
Everything looks good!

**Review by waffy1901 — APPROVED** [review]
Great work!

**Review by solomonmcharo28 — APPROVED** [review]
Great Work John!

**Pull request merged**